src/js/components/Tools.jsx

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var React = require('react');
+
+// Splitting each sidebar into its column
+var Tools = React.createClass({
+  classNames: 'toolbar',
+  render: function() {
+    return (
+      <div className={this.classNames}>
+        <ul>
+          <li>RECT</li>
+          <li>SYMBOL</li>
+          <li>AREA</li>
+          <li>LINE</li>
+          <li>TEXT</li>
+          <li>|</li>
+          <li>EXPORT</li>
+          <li>|</li>
+          <li><i className="fa fa-file-o"></i> CLEAR</li>
+          <li><i className="fa fa-undo"></i> UNDO</li>
+          <li><i className="fa fa-repeat"></i> REDO</li>
```

Comment:
it's all getting redone when we add the new stuff. I totally agree, I just needed to quickly show it it